tests/test_units.py

```diff
@@ -283,3 +283,33 @@ def dryness_index(
             freq: str = "YS",
         ) -> xr.DataArray:
             pass
+
+
+def test_declare_units_partial():
+    def index(data: xr.DataArray, thresh: Quantified, dthreshdt: Quantified):
+        return xr.DataArray(1, attrs={"units": "rad"})
+
+    index_partial = declare_units(
+        partial=True, thresh="<data>", dthreshdt="<data>/[time]"
+    )(index)
+    assert hasattr(index_partial, "_partial_units")
+
+    index_full_mm = declare_units(data="mm")(index_partial)
+    assert index_full_mm.in_units == {
+        "data": "mm",
+        "thresh": "(mm)",
```

Comment:
As relative units can be arithmetical expressions, it's safer to isolated them in parenthesis, just in case they include a space. So when relative units are absolutified, the code always wraps them.